### DISCUSSION TITLE
Search outside of the app from websites like fantasticfiction to see if new books are available.

### DISCUSSION THREAD

**Discussion by cjbrace** [advplyr/audiobookshelf#1227] [Ideas]
Hi, I'm loving this app but I'm finding myself having to leave the app a lot to look for other books I don't have so I can find what I need. 

Would it be possible to add a way to search a service for other books by the Author or Series or just send me to the fantasticfiction or audible page for the author so I can see everything the author has out?

Ideally it would be awesome if there was a Wishlist you can add those searched books too so I can pick what to get hold of, but even just sending us out to another site directly to the authors page would be super useful. 

Thanks for the awesome work!

**Comment by susan3333333** [comment]
An addition to this could be "ghost books" to fill in books missing in a series, it would make it easy to see what books you still needed.
